conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -327,22 +344,24 @@ func (algodImp *algodImporter) Init(ctx context.Context, initProvider data.InitP
 		return nil, fmt.Errorf("unable to fetch genesis file from API at %s", algodImp.cfg.NetAddr)
 	}
 
-	catchpoint := ""
+	if algodImp.needsCatchup(uint64(initProvider.NextDBRound())) {
```

Comment:
```suggestion
	if !algodImp.needsCatchup(uint64(initProvider.NextDBRound())) {
	    return &genesis, err
	}
```

nit: I prefer avoiding so many levels of nesting.